kubectl.fish

```diff
@@ -338,3 +353,7 @@ complete -c kubectl -f -n '__fish_kubectl_needs_command' -a version -d 'Print th
 complete -c kubectl -A -f -n '__fish_seen_subcommand_from version' -l client -d 'Client version only (no server required)'
 complete -c kubectl -A -f -n '__fish_seen_subcommand_from version' -s o -l output -a 'yaml json' -d 'Specify output format'
 complete -c kubectl -A -f -n '__fish_seen_subcommand_from version' -l short -a 'true false' -d 'Print just the version number'
+
+# config
+complete -c kubectl -f -n "__fish_kubectl_using_command config; and not __fish_seen_subcommand_from $__kubectl_config_subcommands" -a '$__kubectl_config_subcommands' -d 'kube config subcommand'
+complete -c kubectl -f -n '__fish_kubectl_using_command config; and __fish_seen_subcommand_from use-context delete-context' -a '(kubectl config get-contexts --no-headers | awk \'{print $2}\')'
```

Comment:
could you replace `-a '(kubectl config get-contexts --no-headers | awk \'{print $2}\')'` with `-a '(kubectl config get-contexts -o name)'`? It prevents having to call `awk`